ansible/roles/test/tasks/pfc_wd/functional_test/ignore_pfc_wd_messages

```diff
@@ -7,3 +7,4 @@ r, ".* snmp.*"
 r, ".* Trying to remove nonexisting queue from flex counter .*"
 r, ".* SAI_STATUS_BUFFER_OVERFLOW"
 r, ".* ERR ntpd.*routing socket reports: No buffer space available.*"
+r, ".* ERR syncd.*"
```

Comment:
Catches following messages on Mellanox platform:
```
        "Oct  3 10:29:19.024127 r-boxer-sw01 INFO syncd#supervisord: syncd Oct 03 10:29:19 ERROR   HOST_INTERFACE: HOST_IFC: policer_id is not enabled for trap_group [0].",
        "Oct  3 10:29:19.024170 r-boxer-sw01 INFO syncd#supervisord: syncd Oct 03 10:29:19 ERROR   HOST_INTERFACE: Failed in host_ifc_policer_bind_get [trap_group = 0], return value: [Entry Not Found]",
```

Those errors are expected, could you add here ```r, ".*syncd.*ERROR HOST_INTERFACE"```?

Reply:
We will probably fix that in newer SAI releases, but for 201811 it is easier to add exception messages in this file

Reply:
Is it three space between `ERROR` and `HOST_INTERFACE` or 1 space?

Reply:
use ` +` to match 1 and more spaces